src/PointwiseFunctions/AnalyticData/GrMhd/SphericalTorus.cpp

```diff
@@ -58,8 +71,9 @@ tnsr::I<T, 3> SphericalTorus::operator()(
     const tnsr::I<T, 3>& source_coords) const {
   tnsr::I<T, 3> result;
   const auto r = radius(get<0>(source_coords));
-  const auto theta =
-      M_PI_2 - (pi_over_2_minus_theta_min_ * get<1>(source_coords));
+  // intermediate variable for equatorial compression
+  const auto intermediate_val = cubic_compression(get<1>(source_coords));
```

Comment:
Having a variable called `intermediate_val` that's only used once isn't adding clarity.  Just call the function in the expression on the next line.

Reply:
okay.